### ISSUE TITLE
Updating the database sqlcachedependency support?

### ISSUE THREAD

**Issue by dotnetshadow** [xivSolutions/biggy#29]
My understanding is in order for the data to be in sync with say SQL Server all CRUDs need to be done using biggy? 

If I update say the "Artists" table directly using sql management studio or using entity framework then Biggy won't be synced correct?

Currently I use sqlcachedependency for tables to ensure that data is refreshed because that data could be modified by other programs

**Comment by xivSolutions** [comment]
Yeah, the in-memory list won't be aware of changes made from another source to the Db. Might be interesting to see if there is a way to notify the Biggy instance (Pub/Sub of some sort?) that the underlying store has changed... thoughts?

Biggy will, of course, be updated the next time it is spun up. If nothing else, maybe we need to add an easy way to poll for changes.

**Comment by dotnetshadow** [comment]
Yes that would be great if the Biggy instance could be notified. I haven't checked the documentation properly but is there a refresh method? or do we just spin a new instance to get the latest data?

Polling is not a bad idea but I know with sqlcachedependency it caused a notification to be sent, I'm not 100% sure if there is an event that shows when something has changed:
http://stackoverflow.com/questions/8486325/sqlcachedependency-does-not-seem-to-have-onchange-event
